roles/openshift_daemonset_config/templates/daemonset.yml.j2

```diff
@@ -101,9 +101,8 @@ spec:
           subPath: sys
           name: host
           readOnly: true
-        - mountPath: /var/run/docker.sock
-          subPath: var/run/docker.sock
-          name: host
+        - mountPath: /run/docker.sock
```

Comment:
I expected `/var/run/docker.sock` here

Reply:
same as above.  /var/run is a symlink to /run.

I can change it to /var/run if that's preferred.

Reply:
if it works, I don't care, just noting it wasn't identical